content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -203,13 +203,11 @@ Defaults to false.
 
 {{< caution >}}
 Executions that are suspended during their scheduled time count as missed jobs.
-When `.spec.suspend` changes from `true` to `false` on an existing cron job without a [starting deadline](#starting-deadline), the missed jobs are scheduled immediately.
+When `.spec.suspend` changes from `true` to `false` on an existing CronJob without a [starting deadline](#starting-deadline), the missed jobs are scheduled immediately.
 {{< /caution >}}
 
-### Jobs History Limits
+### Job History Limits {#job-history-limits}
```

Comment:
I think `Jobs` would be more appropriate here.

E.g., `successfulJobsHistoryLimit` and `failedJobsHistoryLimit` use `Jobs` instead of `Job`.